src/bindings/xfconf/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+include (FindXfconf)
+
+if (NOT XFCONF_FOUND)
+	exclude_binding (xfconf "xfconf >= 4.16 required for xfconf backend")
```

Comment:
Since you require a specific version, is this version actually checked anywhere?

Reply:
No, it was only the version I started with. To be honest, I did not check if there is actually something which requires at least version 4.16. Since 4.16. was released back in december 2020, I thought that this would be sufficient.

Reply:
Even Debian stable has 4.16.0-2 so it should be fine.